test/testserver.js

```diff
@@ -11,6 +11,7 @@ const PORT = 60321;
 let myapi = new API({
   title:        'Test API Server',
   description:  'API server for testing',
+  name:         'auth',
```

Comment:
wrong, this is a different API, not that it matters... name could be `testservice`

Reply:
yes it is merged on other PR. will remove it. :)